### ISSUE TITLE
New command suggestion!

### ISSUE THREAD

**Issue by zSteley** [Bobcat00/ViaVersionStatus#4]
Hi, i can't find discussion/suggestions area in github so I open a issue, sorry for that!

It will be nice a new command, /version that show the info that you get on join! 

Thanks

**Comment by Bobcat00** [comment]
ViaVersion provides a command to show the versions of all currently online players: `/viaversion list`
ProtocolSupport has a similar command: `/ps list`

**Comment by zSteley** [comment]
No, I mean a command for players, that they can see if they are in the correct version

**Comment by zSteley** [comment]
Maybe is possible to add for future versions :D ¿?

**Comment by zSteley** [comment]
¿?

**Comment by mrfloris** [comment]
Can't you just notify the player on join and customize the message to your liking, and if they wish to know, they can relog to see if they are on newer or older, and you can tell them in a custom message what the server is on and to use that instead of the best experience?
